docs/SPEC.md

```diff
@@ -563,13 +563,9 @@ Literals construct a value.
 
 Object literals construct a value with the object type.
 
-    ObjectLiteral = "{" PropertyList "}" .
+    ObjectLiteral = "{" PropertyList | IdentList "}" .
```

Comment:
You have to turn this alternation into another production because the operator precedence implies this is a left brace and a property list or an identity list followed by a right brace.